### PR TITLE
In discover, pick `develop` as a fixed root if `master` isn't present

### PR DESCRIPTION
**Pull request by PawelLipski**
For tests to pass, CLI v2.15.5 (with the same changes applied) must be released

### CODE REVIEW

**Comment by mkondratek** [comment]
Hmmm... :thinking: it looks like something that should be mentioned somehow to the user (by event log?), does it not?
Ideally, we could provide a way to specify **main** branch(es) (custom settings tab #270)

**Comment by PawelLipski** [comment]
Hmm @mkondratek I don't think presenting that detail is so important :thinking: anyway, how would the proposed message go?

**Comment by mkondratek** [comment]
> Hmm @mkondratek I don't think presenting that detail is so important  anyway, how would the proposed message go?

Okay then. It could be smth like `Taking (master|develop) as a presumed root` but let's not get to orthodox about notifying "everything".

**Comment by PawelLipski** [comment]
@mkondratek Hmm yeah the problem is that discover might as well deem some other branches as roots. 
`master` (or `develop`, if `master` missing) are _fixed roots_ aka _roots no matter what_... but other branches may also be considered roots if there's no clear parent to attach them under.
So... I don't what to expose too many details to the user, the discover algorithm is already convoluted enough (and yet still far from perfect).

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,6 +33,12 @@ jobs:
       - run:
           name: Compile tests
           command: ./gradlew compileTestJava
+      - run:
+          name: '[TEMP] Upgrade git-machete CLI'
```
Comment (mkondratek) [inline .circleci/config.yml:37]: What is to be done to remove that? We need to rebuild and push the docker image, right?

Comment (PawelLipski) [reply .circleci/config.yml:37]: image

**Review by micpiotrowski — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (micpiotrowski) [reply .circleci/config.yml:37]: Image regenerated as `gitmachete/intellij-plugin-ci:2.0.0`

**Pull request merged**